src/millie.nim

```diff
@@ -1,7 +1,7 @@
 import millie/utils
-import strutils, strformat, math
+import std/[strutils, math]
 
-const units = @[
```

Comment:
Not a big deal, but since we don't need to change it, we can make it an array instead of a seq

Reply:
I didn't know Nim had arrays! Thanks.